composer.json

```diff
@@ -55,7 +55,8 @@
         "twbs/bootstrap": "^5.3",
         "twbs/bootstrap-icons": "^1.11",
         "webklex/composer-info": "^0.0.1",
-        "zbateson/mail-mime-parser": "^2.4"
+        "zbateson/mail-mime-parser": "^2.4",
+        "garethp/php-ews": "^0.10.1"
```

Comment:
alphabetical would be better.

Reply:
Done. Also switched to latest master upstream. We need this to be merged before using the full set of features developed in this MR: https://github.com/Garethp/php-ews/pull/266